### ISSUE TITLE
shrinking lung syndrome

### ISSUE THREAD

**Issue by anastasia-coding** [obophenotype/human-phenotype-ontology#11346]
**Preferred term label:**
shrinking lung syndrome

**Synonyms**
--------

**Definition (free text, please give PubMed ID)**
Shrinking lung syndrome is characterised by dyspnoea, pleuritic chest pain, and reduced lung volumes. (PMID: 31137062)

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology HP:0002088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Systemic lupus erythematosus (OMIM 152700)

**Your nano-attribution (ORCID)**
Anastasia-Vasiliki Madenidou (https://orcid.org/0000-0002-7382-9958)

**Comment by pnrobinson** [comment]
@anastasia-coding 
I think this is nore of a disease entity (it bundles several items together).
There are codes for reduced lung volume and restrictive ventilatory defects here: Abnormality on pulmonary function testing HP:0030878
with which the individual phenotypic features can be coded. I would suggest this should be an ORPHA or a Mondo entry which could be annotated with HPO terms for the phenotypic spectrum.

**Comment by pnrobinson** [comment]
Closing, I think this is not in scope for HPO.
